### PR TITLE
Refactoring Code

### LINKED ISSUE #513: Button component resulting in undefined after build in the browser.
I got the react-toolbox and dependencies to build after a lot of issues, but once I got it built, when I check the test page in the browser which uses a button I'm getting the error in the console on the following screenshot. Probably have to copy and paste the link, for some reason it always gives a github 404.

http://i.imgur.com/8PR3IRv.png

### PR DESCRIPTION
**Pull request by KerenChandran**
Closes react-toolbox/react-toolbox#513

**Note: When you merge this, please see if the issue above is closed, otherwise please close it to keep that repo clean. I am also going to request Javi to close the issue, so if you have an issue, you should be able to re-open the issue and make comments.**

I finished refactoring your project. I left some comments in the .scss files so you should definitely give that a read. Also you should remove the following files if you're not using it:
- #styles.scss
- bundle.js
- index.js~
- package.json.backup
- package.json~
- react-toolbox.css
- js/components/commons.scss

### CODE REVIEW

File: node/style.scss [hunk 1]
```diff
@@ -1,163 +1,3 @@
...
-
-.dropdownTemplateContent {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 2;
-}
\ No newline at end of file
+@import "~react-toolbox/lib/commons";
+// This file could be used as an Application wide style?
+// You're existing layout is confusing to me, but I hopefully you can refactor this stuff :D
```
Comment (steviesama) [inline node/style.scss:3]: Yeah sorry. I just started with React.js and then stumbled into React Toolbox. I'm hacking all this stuff together trying to figure it out. I haven't remotely tried to organize any of this stuff. Some of it was lifted from different places. Thanks for your help. I'm processing the pull request now and I'll give it a try after I sort it out.

Comment (KerenChandran) [inline node/style.scss:3]: Sure that's cool, take your time!

**Comment by steviesama** [comment]
I merged it and pulled it. It definitely did the trick. Thanks. I'm thinking about all the changes...was it the survivejs kanban presets that fixed it, or the total of the changes you made? I noticed you moved the ./style import to the top and seemed like maybe that could have affected it somehow, but I'm unsure. I'd just been incrementally making changes to get it figured out. I made a lot of progress, but this last one was stumping me. Originally I couldn't get any of it to work, lol.

**Comment by KerenChandran** [comment]
Ya so I couldn't build the project due to the survivejs kanban. Even after that, I was still getting an issue with the CSS, so I looked at the code and saw the css being copied from React Toolbox and stored locally. I wanted to refactor that and only have the styles that are used in the project and reference React Toolbox's styles, so if any new changes happen, you don't have to copy and paste the entire file again. To be honest, I didn't test build once I properly set up React Toolbox to load the styles, and at that point it was fixed.

**Comment by steviesama** [comment]
Ok, so I guess my eyes just weren't following some of the color coding. I see you removed all those entries from the style.scss. I didn't write all of those, I somehow got those from the original repo (or some other) when I was trying to manually set up a bare bones project. Did I inadvertently cause a conflict with those 160'ish lines you removed from it?

**Comment by steviesama** [comment]
Heck, I don't even remember where I got those styles from originally really, I just remember I thought it was necessary for the toolbox.

**Comment by KerenChandran** [comment]
No you didn't. I removed it cause I think you got them from the react toolbox styles. I'm pretty sure I kept all the styles from the react-toolbox example repo. Also, I'm pretty sure there's no need to include those styles, cause it could have been specific overrides for the repo that was using it.

**Comment by steviesama** [comment]
Oh I see...well I think I basically understand what went wrong there. Thanks again. xD

**Comment by KerenChandran** [comment]
Lol no problem! Its been a while since I've contributed to React Toolbox and on Github, so I had a blast! 😄

**Pull request merged**